style.css

```diff
@@ -807,523 +28,772 @@
-    top: 216px;
-    left: 930px;
+  .css-txt {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    font-family: "Poppins", sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    height: 24px;
+    top: 10px;
+    left: 70px;
+    color: #253858;
   }
 
-  .css-txt {
-    margin-top: 70px;
-    margin-left: -60px;
+  .css-frame {
+    position: absolute;
+    width: 18.9rem;
+    height: 4.16rem;
+    top: 890px;
+    left: 25px;
+    list-style: none;
+    border-radius: 8px;
+    background-color: #f7f7f9;
   }
 
   .framewrk {
-    width: 40.5rem;
-    margin-left: 75px;
-    margin-top: 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    list-style: none;
+    width: 20.4rem;
+    height: 4.5rem;
   }
 
   .frmwk-txt {
-    top: 350px;
-    left: 650px;
+    display: flex;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    list-style: none;
+    position: absolute;
+    top: 1010px;
+    height: 24px;
+    left: 30px;
+    right: 78px;
+    color: #000;
   }
 
   .arrow {
-    top: 380px;
-    right: 145px;
+    position: absolute;
+    top: 1035px;
+    right: 66px;
   }
 
   .separator {
-    top: 400px;
-    left: 630px;
+    position: absolute;
+    list-style: none;
+    width: 20.4rem;
+    height: 0.1rem;
+    color: #dfe1e6;
+    top: 1070px;
+    left: 30px;
   }
 
   .SkillS {
-    width: 40.5rem;
-    margin-left: 75px;
-    margin-top: -20px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    list-style: none;
+    width: 327px;
+    height: 72px;
   }
 
   .skill-txt {
-    top: 420px;
-    left: 650px;
+    display: flex;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    list-style: none;
+    position: absolute;
+    height: 24px;
+    top: 1090px;
+    left: 30px;
+    right: 78px;
+    color: #000;
   }
 
   .arrow2 {
-    top: 445px;
-    right: 145px;
+    position: absolute;
+    top: 1115px;
+    right: 66px;
   }
 
   .separator2 {
-    top: 475px;
-    left: 630px;
+    position: absolute;
+    list-style: none;
+    width: 375px;
+    height: 1px;
+    color: #dfe1e6;
+    top: 1145px;
+    left: 30px;
   }
 
-  /* ----------------contact form------------------ */
+  /* -----------contact form--------- */
+
   .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
     position: absolute;
-    width: 90rem;
-    height: 52.87rem;
-    top: 4270px;
-    background: #6070ff;
-    border-top-left-radius: 88px;
+    width: 23.4rem;
+    top: 4800px;
+    height: 48.68rem;
+    border-top-left-radius: 50px;
+    background-color: #6070ff;
+    background-image: url(header-shapesmobile.png);
+    list-style: none;
```

Comment:
- [ ] **Good job so far on adding the contact section :clap:** However, It would be better to make a small adjustment to the styling by adjusting the background image to match as much as possible the Figma design 🙏 you may kindly export the original image from Figma, and kindly consider adding a ```background-size: ``` 👍 👍 

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/220100691-c0dd6e74-3cb5-4463-bf88-af2f38d50b5d.png) | ![image](https://user-images.githubusercontent.com/102757126/220100771-ca565538-bc6f-41ae-afa3-53905558cea0.png)